pages/osx/java_home.md

```diff
@@ -0,0 +1,24 @@
+# java_home
+
+> Return a value for $JAVA_HOME or execute command using this variable.
+> More information: <https://www.unix.com/man-page/osx/1/java_home>.
+
+- List JVMs based on a specific version:
+
+`java_home --version {{1.5+}}`
+
+- List JVMs based on a specific [arch]itecture:
+
+`java_home --arch {{i386}}`
+
+- List JVMs based on a specific tasks (default: `CommandLine`):
```

Comment:
```suggestion
- List JVMs based on a specific tasks (defaults to `CommandLine`):
```

Reply:
I think it's a nice idea to prefer verbs over nouns. Later I'll standardize this note about default values.